app/src/main/kotlin/com/hippo/ehviewer/client/EhCookieStore.kt

```diff
@@ -55,16 +28,13 @@ object EhCookieStore : CookieJar {
     private const val KEY_CONTENT_WARNING = "nw"
     private const val CONTENT_WARNING_NOT_SHOW = "1"
     private const val KEY_UTMP_NAME = "__utmp"
-    private val sTipsCookie = Cookie.Builder().apply {
-        name(KEY_CONTENT_WARNING)
-        value(CONTENT_WARNING_NOT_SHOW)
-        domain(EhUrl.DOMAIN_E)
-        path("/")
-        expiresAt(Long.MAX_VALUE)
-    }.build()
+    private val sTipsCookie = Cookie(
+        name = KEY_CONTENT_WARNING,
+        value = CONTENT_WARNING_NOT_SHOW,
+    )
```

Comment:
Should set expires?

Reply:
> Should set expires?

We don't save it in Webkit CookieManager, but add it dynamically when load cookies for url
I think no need

Reply:
Here though https://github.com/FooIbar/EhViewer/blob/5bd0f5f97b3a16cad0c4242fb9922c75618066be/app/src/main/kotlin/com/hippo/ehviewer/client/EhCookieStore.kt#L49-L53

Reply:
Done